tests/vm_test.go

```diff
@@ -154,7 +154,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			return newVM
 		}
 
-		It("should set the default MachineType when created without explicit value", func() {
+		It("[test_id:3312]should set the default MachineType when created without explicit value", func() {
```

Comment:
The test_id here and at line 169 are the same.

Reply:
Fixed